sandy/decay.py

```diff
@@ -860,15 +874,13 @@ def to_hdf5(self, filename, lib, mode="a"):
         {10010: {'decay_constant': 0, 'decay_constant_uncertainty': 0,
                  'decay_energy': {'alpha': 0.0, 'beta': 0.0, 'gamma': 0.0},
                  'decay_energy_uncertainties': {'alpha': 0.0, 'beta': 0.0, 'gamma': 0.0},
-                 'half_life': 0.0, 'half_life_uncertainty': 0.0, 'parity': 1.0, 'spin': 0.5, 'stable': True}, 
-         270600: {'decay_constant': 4.167050502344267e-09, 'decay_constant_uncertainty': 6.324352137605637e-13, 
-                  'decay_energy': {'alpha': 0.0, 'beta': 96522.0, 'gamma': 2503840.0}, 
-                  'decay_energy_uncertainties': {'alpha': 0.0, 'beta': 202.529, 'gamma': 352.186},
-                  'decay_modes': {10: {'branching_ratio': 1.0, 'branching_ratio_uncertainty': 0.0, 'decay_products': {280600: 1.0}}},
+                 'half_life': 0.0, 'half_life_uncertainty': 0.0, 'parity': 1.0, 'spin': 0.5, 'stable': True},
+         270600: {'decay_constant': 4.167050502344267e-09, 'decay_constant_uncertainty': 6.324352137605637e-13,
+                  'decay_energy': {'alpha': 0.0, 'beta': 96522.0, 'gamma': 2503840.0}, 'decay_energy_uncertainties': {'alpha': 0.0, 'beta': 202.529, 'gamma': 352.186},
+                  'decay_modes': {"('10', 0)": {'branching_ratio': 1.0, 'branching_ratio_uncertainty': 0.0, 'decay_products': {280600: 1.0}}},
```

Comment:
In tuple `"('10', 0)"` the first key is a string and the second is an integer... can we do something better of should we leave it as it is?

Reply:
I prefer both  integers